src/call/client.rs

```diff
@@ -167,7 +167,7 @@ impl Call {
         mut opt: CallOption,
     ) -> Result<ClientSStreamReceiver<Resp>> {
         let call = channel.create_call(method, &opt)?;
-        let mut payload = vec![];
+        let mut payload = Vec::new();
```

Comment:
Allocating 0-sized memory because we'll reserve a size later and we don't need the first allocation.

Ref: https://stackoverflow.com/q/2022335/7083401